docs/standalone/audit_guidelines.md

```diff
@@ -0,0 +1,137 @@
+* Execution cost limits;
+* Double satisfaction;
+* Replay;
+* Denial of service.
+
+The outcome of the review could also include, without being limited to:
+* a validation that the Hydra plutus scripts validators do check the transaction constraints defined in the Coordinated Hydra Head V1 specification;
+* a review and comment on the mutation-based tests applied to the Hydra plutus scripts and, in particular, any adversarial situation that would not be covered by them but should be.
+
+See the documentation of our [Mutation-Based tests](https://hydra.family/head-protocol/haddock/hydra-node/tests/Hydra-Chain-Direct-Contract-Mutation.html).
+
+See [Common Weaknesses](https://plutus.readthedocs.io/en/latest/reference/writing-scripts/common-weaknesses/index.html) and [Vulnerabilities](https://github.com/Plutonomicon/plutonomicon/blob/main/vulnerabilities.md).
+
+## Hydra node chain layer code generates transactions which are consistent with Hydra Head V1 specification
+
+The Coordinated Hydra Head V1 specification defines the transactions the off-chain code should build and post to evolve the head status on-chain.
+
+You could assess that the Hydra node chain layer code can only build transactions which are consistent with the Coordinated Hydra Head V1 specification.
+
+# Out of Scope
+
+With version 0.9, the Hydra team suggests to focus on the scope described in the above sections and consider anything else as out of scope.
+In particular, the following items shoudl be seen as out of scope:
+
+* Hydra Head protocol implementation is immune to attacks via chain transactions.
+* Hydra Head protocol implementation is immune to attacks via network.
+* Hydra Head protocol implementation faithfully reflects the head state through the API.
+* Hydra Head protocol implementation is immune to API attacks.
+* Any attack which would be invalid under the assumptions stated above in the document.
+
+# Artifacts
+
+This sections gives a detailed description of the artifacts mentioned above in the document:
+ - Coordinated Hydra Head V1 Specification
+ - Hydra plutus scripts (on-chain code)
+ - Hydra node chain layer code (off-chain code)
+
+## Artifact 1: Coordinated Hydra Head V1 Specification
+
+The Hydra Head protocol implementation derives from [Hydra: Fast Isomorphic State Channels](https://eprint.iacr.org/2020/299.pdf) in several ways. Especially some simplifications have been introduced and generalizations removed.
+
+The [Coordinated Hydra Head V1 specification](https://docs.google.com/document/d/1XQ0C7Ko3Ifo5a4TOcW1fDT8gMYryB54PCEgOiFaAwGE/) captures these deviations and also includes the "formal notation" of the actual transaction constraints (which are foregone in the original paper). Also, it details the L2 protocol logic for the **Coordinated** Head protocol - which is implemented in V1.
```

Comment:
@ch1bo do you have a better link to the new public specification already? Otherwise we keep it that way for know and will update it later.

Reply:
Yes, definitely. Let's rather link the overleaf (viewing rights): https://www.overleaf.com/read/bbqzmptcxryj

With #693 we will have it in a better place and hopefully also documented how to suggest changes.